docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -13,42 +13,53 @@ include::_attributes.adoc[]
 :topics: TLS, http, SSL, HTTPS, security, network
 :extensions: io.quarkus:quarkus-tls-registry
 
-The TLS registry is a Quarkus extension centralizing the TLS configuration for the application.
-You can use the TLS registry to define the configuration in one place and reference it from multiple places in the application.
+The TLS Registry is a Quarkus extension that centralizes TLS configuration, making it easier to manage and maintain secure connections across your application.
+By defining TLS configurations in a single, centralized location, the TLS Registry allows you to reference these configurations from multiple components within the application, ensuring consistency and reducing the potential for configuration errors.
 
-The TLS extension is automatically added to your project when you use a compatible extension.
-For example, if your application uses Quarkus REST, gRPC, or reactive routes, the TLS registry is automatically added to your project.
+The TLS Registry consolidates settings and supports multiple named configurations, enabling you to tailor TLS settings for different application parts.
+This flexibility is particularly useful when different components require distinct security configurations.
 
-== Using TLS registry
+Additionally, the TLS Registry extension is automatically included in your project when you use compatible extensions such as Quarkus REST, gRPC, or Reactive Routes, enabling applications to be ready to handle secure communications out of the box.
+TLS Registry also provides features like automatic certificate reloading, Let's Encrypt (ACME) integration, Kubernetes Cert-Manager support, and compatibility with various keystore formats, such as PKCS12, PEM, and JKS.
 
-To configure a TLS connection, including key and trust stores, use the `quarkus.tls.*` properties.
+[#using-the-tls-registry]
+== Using the TLS registry
 
-The configuration model always contains a default (unnamed) TLS configuration that is configured by properties directly under `quarkus.tls.*`.
-In addition, it allows you to define separate named configurations; these use `quarkus.tls.<name>.*` properties.
-However, by using the `quarkus.tls.<name>.*` properties, you can also have specific configurations for specific connections.
+To configure a TLS connection, including key and trust stores, use the `+quarkus.tls.*+` properties.
+These properties are required for setting up the default TLS configuration, defined directly under `+quarkus.tls.*+`, and for creating separate, named configurations by using `+quarkus.tls.<name>.*+`.
+By specifying `+quarkus.tls.<name>.*+` properties, you can adapt TLS settings for specific connections.
```

Comment:
it's not really a "connection" (I would avoid that word because it means something different when we say TLS connection). I would use "component".

Reply:
OK! Applying :).